app/helpers/images_helper.rb

```diff
@@ -103,15 +103,6 @@ def show_image_copyright?(image, object)
        image.copyright_holder != object.user&.legal_name)
   end
 
-  def show_best_image(obs)
```

Comment:
`show_best_image` looks like dead code.
Is there a reason to keep it (or am I wrong about it being dead.)

Reply:
@JoeCohen I had asked Nathan to keep it with a comment that it's for getting the top image in a matrix box carousel, which i think we'd like to re-enable after Bootstrap update. 

I think he forgot to add the comment.

Reply:
Thanks so much! I'll restore it and add a comment.

Reply:
@JoeCohen I could be wrong! see my revised comment above

Reply:
@JoeCohen  Okay, i believe this is dead code after all. 

The way we find obs images involves batch-getting, then a re-sort that puts the thumb_image on top. This method is indeed obsolete then.

Reply:
@nimmolo. Thanks for the clarifications. I won't restore it.